src/main/java/com/basilisk/frontend/components/TweetDisplayComponent.java

```diff
@@ -32,6 +32,10 @@
     private Button dislikeButton;
     @Id("commentButton")
     private Button commentButton;
```

Comment:
Why use two buttons? Why can you just have 1 button and switch the text between the buttons?

Reply:
I'm gonna go ahead and approve this for now since you probably have a valid reasoning for it

Reply:
The reason for the usage of 2 buttons is that I thought that we should allow the user to expand the comment section / show more comments, all the while giving the user the ability to hide the comments, even if not all of them are being shown. Picture of the explanation below.

https://media.discordapp.net/attachments/533331244277039116/556291442377490448/unknown.png?width=700&height=638